pleskdistup/common/src/plesk.py

```diff
@@ -124,10 +127,16 @@ def list_installed_extensions() -> typing.List[typing.Tuple[str, str]]:
 
 
 def install_extension(name: str) -> None:
+    if not is_plesk_database_ready():
```

Comment:
Note that this checks only if the service is up, not that it can service any requests (e.g. authentication works correctly and DB is alive).

Reply:
The change mainly ensures there are no problems on `revert`, as we know the database service has been stopped on preparation stage.
So far, we haven't had any issues with incorrect authentication or database corruption. If such problems do occur, a thorough investigation would be needed. It's likely to cause the entire conversion process to fail, so skipping some conversion steps doesn't make much sense. So yeah, it's kind of expected for this change,

Reply:
OK